Explorer/Assets/DCL/SDKComponents/SceneUI/Systems/UIDropdown/UIDropdownReleaseSystem.cs

```diff
@@ -24,7 +24,7 @@ public partial class UIDropdownReleaseSystem : BaseUnityLoopSystem
 
         private UIDropdownReleaseSystem(World world, IComponentPoolsRegistry poolsRegistry) : base(world)
         {
-            poolsRegistry.TryGetPool(typeof(DCLDropdown), out componentPool);
+            poolsRegistry.TryGetPool(typeof(UIDropdownComponent), out componentPool);
```

Comment:
I think you can use `GetReferenceTypePool<T>` instead and don't expect it to return `null`. In general if you didn't register the pool of this type for some reason it should be detected as soon as possible and not linger silently

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/303/commits/56cc6f24969c036b6363ad201a03c416618f94a1